### PR TITLE
Add new workflow example demonstrating how assets are created and mac…

### PR DESCRIPTION
**Pull request by paulgconlon**
…hine data pushed to Sitelink3D.

### CODE REVIEW

File: components/datalogger/datalogger_utils.py [hunk 1]
```diff
@@ -714,3 +721,112 @@ def ProcessDataloggerToCsv(a_server, a_site_id, a_headers, a_target_dir, a_datal
...
+    manifest.write(struct.pack("<d", latitude))
+    manifest.write(struct.pack("<d", longitude))
+    manifest.write(struct.pack("<d", altitude))
+    manifest.write(struct.pack("<f", direction))
+    manifest.seek(0)
+    return {
+        "type": "mfk::Replicate",
+        "data": {
+            "ac_uuid": asset_context.uuid,
+            "beacon": True,
```
Comment (topcon-lrietberg) [inline components/datalogger/datalogger_utils.py:821]: beacon flag should be set according to timestamp

File: workflows/machine_data/live/write/create_and_connect_machine_to_site/create_and_connect_machine_to_site.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Authorization. OAuth credentials are used if the JWT string is empty.
+## run `SitelinkFrontend.core.store.getState().app.owner.jwt[0]` in your browser developer console to obtain a JWT.
+jwt=""
+# - or -
+oauth_id=""
+oauth_secret=""
+oauth_scope=""
+
+machine_resource_configuration_file="resource_configurations/haul_truck.resource_configuration.json"
```
Comment (topcon-lrietberg) [inline workflows/machine_data/live/write/create_and_connect_machine_to_site/create_and_connect_machine_to_site.sh:17]: where is haul_truck.resource_configuration.json?, I would like to confirm it is using wgs for replicate transforms as it is indicated in the mfk replicate function make_haul_mfk_replicate_payload.

File: components/datalogger/datalogger_utils.py (hunk 1 above)
Comment (paulgconlon) [reply components/datalogger/datalogger_utils.py]: Thanks, have implemented for low frequency beacon.

File: workflows/machine_data/live/write/create_and_connect_machine_to_site/create_and_connect_machine_to_site.sh (hunk 2 above)
Comment (paulgconlon) [reply workflows/machine_data/live/write/create_and_connect_machine_to_site/create_and_connect_machine_to_site.sh:17]: Sorry, missed commit of the resource configuration.

**Review by topcon-lrietberg — APPROVED** [review]

**Pull request merged**